src/utils/discord.js

```diff
@@ -114,7 +114,7 @@ export const validateMessage = message => {
     tts: Boolean(message.tts),
     flags: validateKeys(message.flags || message, INTERACTION_RESPONSE_FLAGS),
```

Comment:
Might be able to rely on the API to sanitize flags for us:
 - https://github.com/discordjs/discord.js/blob/main/src/structures/MessagePayload.js#L153-L155